extras/org.locationtech.udig.target/org.locationtech.udig.target.target

```diff
@@ -127,6 +127,10 @@
 <unit id="javax.xml.bind" version="0.0.0"/>
 <unit id="javax.xml.stream" version="0.0.0"/>
 <unit id="javax.activation" version="1.1.0.v201211130549"/>
+<unit id="javax.activation" version="1.2.2.v20201119-1642"/>
```

Comment:
Do we need javax.activation twice?

Reply:
Good point. the dependency tree in libs-bundle (pom-libs) listed it as a transitive dependency to jaxb-api (2.4x). 

I can check it again, if it is required with jaxb 2.2. To be honest: I coundn't find jaxb-api in Eclipse universe (e.g. Orbit repositories) yet.  Since I tested imagemosaic (requires jaxb) with shapefiles successfully I assume jaxb 2.2 is fine.